<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snowgui::repositorywindow</class>
 <widget class="QWidget" name="snowgui::repositorywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="repositoryTree">
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="columnCount">
      <number>13</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">6</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">7</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">8</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">9</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">10</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">11</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">12</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">13</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="previewButton">
        <property name="text">
         <string>Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
